src/io.cr

```diff
@@ -81,6 +81,7 @@ abstract class IO
   #
   # ```
   # STDIN.read_timeout = 1
+  # sleep 1
```

Comment:
This is correct as-is, an `IO::Timeout` is raised when the user didn't enter something at the `gets` after 1 second. Adding this sleep would make the delay 2 seconds.
```suggestion
```

Reply:
@r00ster91 Oh, you are completely right. I understood my misunderstanding. Thank you for the nice point! 😺